### ISSUE TITLE
Order summary title colour is not visible (colour contrast issue)

### ISSUE THREAD

**Issue by chrisjsimpson** [Subscribie/subscribie#844]
**Is your feature request related to a problem? Please describe.**

See 'Order summary' on right:

![image](https://user-images.githubusercontent.com/1718624/161386988-41ca29b1-9209-41ff-add9-066a180c7532.png)

**Describe the solution you'd like**

Use existing css classes to correct this

**Describe alternatives you've considered**

**Additional context**
